src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -1399,21 +1420,80 @@ impl_commit_tree (RpmOstreeTreeComposeContext *self,
       if (!ostree_repo_commit_transaction (self->repo, &stats, cancellable, error))
         return glnx_prefix_error (error, "Commit");
 
+      g_variant_builder_add (&composemeta_builder, "{sv}", "ostree-n-metadata-total",
+                             g_variant_new_uint32 (stats.metadata_objects_total));
       g_print ("Metadata Total: %u\n", stats.metadata_objects_total);
+
+      g_variant_builder_add (&composemeta_builder, "{sv}", "ostree-n-metadata-written",
+                             g_variant_new_uint32 (stats.metadata_objects_written));
       g_print ("Metadata Written: %u\n", stats.metadata_objects_written);
+
+      g_variant_builder_add (&composemeta_builder, "{sv}", "ostree-n-content-total",
+                             g_variant_new_uint32 (stats.content_objects_total));
       g_print ("Content Total: %u\n", stats.content_objects_total);
+
       g_print ("Content Written: %u\n", stats.content_objects_written);
+      g_variant_builder_add (&composemeta_builder, "{sv}", "ostree-n-content-written",
+                             g_variant_new_uint32 (stats.content_objects_written));
+
       g_print ("Content Bytes Written: %" G_GUINT64_FORMAT "\n", stats.content_bytes_written);
+      g_variant_builder_add (&composemeta_builder, "{sv}", "ostree-content-bytes-written",
+                             g_variant_new_uint64 (stats.content_bytes_written));
     }
   g_print ("Wrote commit: %s\n", new_revision);
+  g_variant_builder_add (&composemeta_builder, "{sv}", "ostree-commit", g_variant_new_string (new_revision));
```

Comment:
Let's also add `ostree-version` here? Especially since it's one of those values that are auto-generated if automatic version prefixing is enabled.